src/uct/api/uct.h

```diff
@@ -337,8 +341,9 @@ enum uct_md_mem_flags {
                                                 mapping may be deferred until
                                                 it is accessed by the CPU or a
                                                 transport. */
-    UCT_MD_MEM_FLAG_FIXED    = UCS_BIT(1)  /**< Place the mapping at exactly
+    UCT_MD_MEM_FLAG_FIXED    = UCS_BIT(1), /**< Place the mapping at exactly
                                                 defined address */
+    UCT_MD_MEM_FLAG_EMPTY    = UCS_BIT(2)  /**< Create empty handle (for UMR) */
```

Comment:
how should it be used?
also UMR is IB-specific terminology which can't be used it uct common code